examples/chart/teleport-agent/values.yaml

```diff
@@ -0,0 +1,36 @@
+##################################################
+# Values that must always be provided by the user.
+##################################################
+
+# Join token for the cluster.
+authToken:
+# Address of the teleport proxy with port (usually :3080).
+proxyAddr:
+# Name for this kubernetes cluster to be used by teleport users.
+kubeClusterName:
+
+##################################################
+# Values that you may need to change.
+##################################################
+
+# Version of teleport image.
+teleportVersion: 5.0.0-rc.2
+# When set to true, the agent will skip the verification of proxy TLS
+# certificate.
+insecureSkipProxyTLSVerify: false
+
+##################################################
+# Values that you shouldn't need to change.
+##################################################
+
+# Container image for the agent. Since this runs without the auth_service, we
+# don't need the enterprise version.
+image: quay.io/gravitational/teleport
+# Number of replicas for the agent deployment.
+replicaCount: 1
+# Name of the ClusterRole, used by the agent's service account.
+clusterRoleName: teleport-agent-cr
```

Comment:
I'd consider getting rid of redundant suffixes `-cr` here and `-crb` below.

Reply:
done